Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -92,23 +91,39 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
       )
       .map(first)
 
-    let project = configData.map { $0.0 }
+    let initialEvent = configData.map(second)
+
+    let updatedEventFetch = initialEvent
+      .switchMap { event -> SignalProducer<Event<LiveStreamEvent, LiveApiError>, NoError> in
+        AppEnvironment.current.liveStreamService
+          .fetchEvent(eventId: event.id, uid: AppEnvironment.current.currentUser?.id)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .materialize()
+    }
 
+    let project = configData.map(first)
     let event = Signal.merge(
-      configData.map { $0.2 }.skipNil(),
-      self.liveStreamEventProperty.signal.skipNil()
+      initialEvent,
+      updatedEventFetch.values()
     )
 
-    self.createAndConfigureLiveStreamViewController = Signal.combineLatest(project, event)
+    self.configureLiveStreamViewController = Signal.combineLatest(project, event.skip(first: 1))
```

Comment:
What's the feeling on using `.skip(first: 1)` here? We know that we only want to configure the live stream view controller once the fresh event has been fetched and our tests confirm this so should be cool?